src/routes/safe/components/Transactions/helpers/TxParametersDetail/index.tsx

```diff
@@ -126,25 +145,32 @@ export const TxParametersDetail = ({
           <StyledText size="md" color="placeHolder">
             Safe transaction parameters
           </StyledText>
-
-          <TxParameterWrapper>
-            <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={color}>
-              Safe nonce
-            </ColoredText>
-            <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={color}>
-              {txParameters.safeNonce}
-            </ColoredText>
-          </TxParameterWrapper>
+          <TxParameter
+            name={
+              <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={color}>
+                Safe nonce
+              </ColoredText>
+            }
+            value={
+              <ColoredText size="lg" isOutOfOrder={isTxNonceOutOfOrder} color={color}>
+                {txParameters.safeNonce}
+              </ColoredText>
+            }
+          />
 
           {showSafeTxGas && (
-            <TxParameterWrapper>
-              <Text size="lg" color={color}>
-                SafeTxGas
-              </Text>
-              <Text size="lg" color={color}>
-                {txParameters.safeTxGas}
-              </Text>
-            </TxParameterWrapper>
+            <TxParameter
+              name={
+                <Text size="lg" color={color}>
```

Comment:
Why pass this and not just `"SafeTxGas"`?

Reply:
Color could be a prop on the TxParameter itself.

Reply:
It takes a custom prop, `color`.

Reply:
Why? The others use the default color as standard.

Reply:
Changed it to accept `color` and `isError`. Please lmk what you think.